domain-models-runtime/src/main/java/org/folio/rest/persist/PgUtil.java

```diff
@@ -1022,6 +1041,8 @@ private static void streamTrailer(HttpServerResponse response, ResultInfo result
       return;
     }
 
+    final Method respond409 = getRespond409(responseClass);
```

Comment:
This should go into the following try block, sorted between respond201 and respond413.
Only if all code is in try blocks we can assure that all exceptions are properly returned via the AsyncResult Handler.

Reply:
> This should go into the following try block, sorted between respond201 and respond413.
> Only if all code is in try blocks we can assure that all exceptions are properly returned via the AsyncResult Handler.

Done.